package.json

```diff
@@ -7,13 +7,16 @@
     "deploy-staging": "sh scripts/deploy-staging.sh",
     "deploy-storybook": "storybook-to-ghpages",
     "deploy": "sh scripts/deploy.sh",
+    "generate-types": "apollo codegen:generate --target typescript --outputFlat src/v2/apollo/types",
+    "generate-types:watch": "yarn generate-types --watch",
```

Comment:
Once more TypeScript is added to / converted in the codebase and graphql queries are being written on the fly, a future workflow item (to avoid starting processes in two separate tabs) would be to add this to package.json scripts:

```
"start-dev": "concurrently --raw --kill-others 'yarn start' 'yarn generate-types:watch'"
```
The server boots, and then apollo codegen is in the background spinning generating types as new stuff comes along or as graphql code changes. VSCode would then pick up the changes.